clients/web/src/views/App.js

```diff
@@ -40,6 +40,7 @@ var App = View.extend({
         this._started = false;
         settings = settings || {};
         this.contactEmail = settings.contactEmail || null;
+        this.brandName = settings.brandName || null;
```

Comment:
This actually can stay as `null`, as long as dependent views have a default fallback of `'Girder'`. Leaving it as `null` preserves the information that an instance name was not set on App instantiation.